Tasks/ContainerStructureTestV0/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 1,
```

Comment:
Any reason we are not changing the minor version as per sprint, like we do in other tasks?

Reply:
no specific reason. May be I should start doing that now.